java/tour-of-restate/src/main/resources/log4j2.properties

```diff
@@ -0,0 +1,18 @@
+# Set to debug or trace if log4j initialization is failing
```

Comment:
Can you replace this with https://github.com/restatedev/sdk-java?tab=readme-ov-file#logging ?

Reply:
Since the Tour in the docs contains logs, I will do this the next time I go through the Tour to do the CLI updates.